src/finch/tensor/tensor.py

```diff
@@ -21,10 +24,10 @@ def __add__(self, other):
     def __mul__(self, other):
         pass
 
+
 def fill_value(arg):
-    if isinstance(arg, LazyTensor):
+    if isinstance(arg, AbstractTensor):
```

Comment:
`LazyTensor` was not imported here.

Reply:
I think fill_value is defined in algebra.py, and this definition is extra and should be moved or migrated. Sorry for the dead code